src/biopsykit/signals/ecg/plotting.py

```diff
@@ -287,33 +289,34 @@ def hr_plot(
     Parameters
     ----------
     heart_rate : :class:`~pandas.DataFrame`
-        Dataframe with heart rate output. Output from :meth:`~biopsykit.signals.ecg.ecg.EcgProcessor.ecg_process()`
+        Dataframe with heart rate output. Output from :meth:`~biopsykit.signals.ecg.ecg.EcgProcessor.ecg_process()`.
     plot_mean : bool, optional
         Whether to plot the mean heart rate as horizontal line or not. Default: ``True``
     plot_outlier : bool, optional
         Whether to plot ECG signal outlier as vertical outlier or not. Default: ``False``
-    outlier : :class:`numpy.array`, optional
+    outlier : :func:`numpy.array`, optional
```

Comment:
@Rebecca243 see above

```suggestion
    outlier : :class:`numpy.ndarray`, optional
```